ciphers/enigma_machine2.py

```diff
@@ -124,7 +124,8 @@ def _plugboard(pbstring: str) -> dict:
     # a) is type string
     # b) has even length (so pairs can be made)
     if not isinstance(pbstring, str):
-        raise TypeError(f"Plugboard setting isn't type string ({type(pbstring)})")
+        raise TypeError(
+            f"Plugboard setting isn't type string ({type(pbstring)})")
```

Comment:
```suggestion
            f"Plugboard setting isn't type string ({type(pbstring)})"
        )
```
This is what psf/black will do.

Reply:
fuuu i need to turn off autoformat on save